### PR TITLE
Local supabase and local OTP

### PR DESCRIPTION
**Pull request by amatlock22**
Adds:
- One-time-passcode (OTP) for local development
  - Utilizes `Inbucket URL` link to manage fake emails
- Additional README information for supabase setup and management

_Note: `supabase` folder was added to `.gitignore`. Not sure if it should be there once we figure out how this all actually works._

### CODE REVIEW

**Review by johnverrone — APPROVED** [review]
noiceeee. lgtm

File: README.md [hunk 1]
```diff
@@ -2,13 +2,58 @@
 
 This app is built using [SvelteKit](https://kit.svelte.dev/) and [TypeScript](https://www.typescriptlang.org/).
 
+## Prerequisites
+
+For development, you need the following installed:
+
+- [Docker](https://docs.docker.com/engine/install/)
+- [Supabase CLI](https://supabase.com/docs/guides/cli)
```
Comment (johnverrone) [inline README.md]: it looks like we can install `supabase` with npm if we want that to live in our `package.json` and not as pre-req dependency: https://supabase.com/docs/guides/cli#installation

and then you'd just run the binary using `npx supabase ...`

Comment (amatlock22) [reply README.md]: Yeah I probably should've done that but I made the decision to brew install it. I can switch to that though.

File: .gitignore [hunk 2]
```diff
@@ -6,3 +6,4 @@ node_modules
 .env
 .env.*
 !.env.example
+supabase/
```
Comment (johnverrone) [inline .gitignore]: looks like this should be committed to VCS so we can run db migrations and stuff during CI for prod.
```suggestion

```

File: README.md (hunk 1 above)
Comment (amatlock22) [reply README.md]: Added in [1a8b60a](https://github.com/johnverrone/bunkie-bois/pull/8/commits/1a8b60a57d697f9f20091afec689af8a4185731e)

File: .gitignore (hunk 2 above)
Comment (amatlock22) [reply .gitignore]: Added in [98728d5](https://github.com/johnverrone/bunkie-bois/pull/8/commits/98728d5bdbdb3656c8a191f08b2ae393c2fe3bdd)

**Pull request merged**